.github/contributing.md

```diff
@@ -0,0 +1,78 @@
+# `too-many-hooks` contributing guide
+
+Thanks for taking the time to contribute to `too-many-hooks`! Any merged PRs will be reflected in the next release of the package and on our storybook immediately.
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md). Here are some resources to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+## Getting started
+
+Anyone can open an issue or PR so if there is something you'd like to see added to `too-many-hooks` feel free to open an issue, PR, or both according to the guidelines below
+
+### Issues
+
+#### Create a new issue
+
+If you find an issue with one of the hooks or stories, [search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments). If a related issue doesn't exist, open a new issue.
+
+When creating an issue ensure the title is in the imperative mood i.e. `Update contributing guide`  and that the description describes the requested feature / steps to reproduce an issue.
+
+#### Claim an issue
+
+Scan through our [existing issues](https://github.com/yobgob/too-many-hooks/issues) to find one that interests you. You can narrow down the search using `labels` as filters. If you find one that has no assignees or has had no updates for a while, leave a comment asking for it to be assigned to you.
+
+Assignees are only used for tracking purposes to prevent redundant work. As long as an issue has no assignees or comments claiming it, you can drop a comment and get started.
```

Comment:
This "only" might not be necessary

```suggestion
Assignees are used for tracking purposes to prevent redundant work. As long as an issue has no assignees or comments claiming it, you can drop a comment and get started.
```